### DISCUSSION TITLE
[@types/react]@18.0.0 No "children" props in FunctionComponent interface

### DISCUSSION THREAD

**Discussion by JakeKey** [DefinitelyTyped/DefinitelyTyped#59992] [Issues with a @types package]
In previous versions **PropsWithChildren** type was used, in 18.0.0 it's defined but not used. That makes a neccessity of declaration _children_ prop for every component manually.

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
This change was intended. Check out https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210 for a full list of intended, breaking changes and their rationale. The PR includes links to a codemod that helps migrating to `@types/react@18`
